app/scripts/controllers/app-state-controller.ts

```diff
@@ -181,7 +188,6 @@ const getDefaultAppStateControllerState = (
   newPrivacyPolicyToastClickedOrClosed: null,
   newPrivacyPolicyToastShownDate: null,
   hadAdvancedGasFeesSetPriorToMigration92_3: false,
-  ...initState,
```

Comment:
All of the properties _after_ this line in the initial state were overwritten with defaults each time the controller was constructed.

To emulate that, I'd recommend that we set them all to `persist: false` and use a migration to delete them. That should ensure they get reset to the default each time, and it will save on disk space.

Reply:
Or we could reset them explicitly upon construction each time. Or both? Saving disk space sounds nice to me though.

Reply:
Sure make sense I will opt in for both